File: 2018/src/test/java/com/benjamin/Day1Test.java
package com.benjamin;

import org.hamcrest.Matchers;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.hamcrest.MatcherAssert.assertThat;

public class Day1Test {

    private Day1 target;

    @BeforeEach
    public void setup() {
        target = new Day1();
    }

    @Test
    public void testDeelEen1() {
        int result = target.deelEenA("");

        assertThat(result, Matchers.is(3));
    }

    @Test
    public void testDeelTwee1() {
        int result = target.deelTweeA("");

        assertThat(result, Matchers.is(6));
    }
}

File: 2021/src/test/kotlin/Day1Test.kt
import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

internal class Day1Test {

    @Test
    fun `should test part1`() {
      val testInput = """199
200
208
210
200
207
240
269
260
263"""

      assertEquals(7, Day1.part1(testInput))
    }
}
